chipsec/utilcmd/uefi_cmd.py

```diff
@@ -72,303 +72,367 @@ class UEFICommand(BaseCommand):
+
+        # keys command args
+        parser_keys = subparsers.add_parser('keys')
+        parser_keys.add_argument('filename', type=str, help='name of file containing variables')
+        parser_keys.set_defaults(func=self.keys)
+
+        # tables command args
+        parser_tables = subparsers.add_parser('tables')
+        parser_tables.set_defaults(func=self.tables)
+
+        # s3bootscript command args
+        parser_bootscript = subparsers.add_parser('s3bootscript')
+        parser_bootscript.set_defaults(func=self.s3bootscript)
+        parser_bootscript.add_argument('bootscript_pa',type=lambda x: int(x,0), nargs='?', help='')
+
+        # insert-before command args
+        parser_insert_before = subparsers.add_parser('insert-before')
+        parser_insert_before.add_argument('guid', type=str, help='guid')
+        parser_insert_before.add_argument('filename', type=str, help='')
+        parser_insert_before.add_argument('new_file', type=str, help='')
+        parser_insert_before.add_argument('efi_file', type=str, help='')
+        parser_insert_before.set_defaults(func=self.insert_before)
+
+        # insert-after command args
+        parser_insert_after = subparsers.add_parser('insert-after')
+        parser_insert_after.add_argument('guid', type=str, help='guid')
+        parser_insert_after.add_argument('filename', type=str, help='')
+        parser_insert_after.add_argument('new_file', type=str, help='')
+        parser_insert_after.add_argument('efi_file', type=str, help='')
+        parser_insert_after.set_defaults(func=self.insert_after)
+
+        # replace command args
+        parser_replace = subparsers.add_parser('replace')
+        parser_replace.add_argument('guid', type=str, help='guid')
+        parser_replace.add_argument('filename', type=str, help='')
+        parser_replace.add_argument('new_file', type=str, help='')
+        parser_replace.add_argument('efi_file', type=str, help='')
+        parser_replace.set_defaults(func=self.replace)
+
+        # remove command args
+        parser_remove = subparsers.add_parser('remove')
+        parser_remove.add_argument('guid', type=str, help='guid')
+        parser_remove.add_argument('filename', type=str, help='')
+        parser_remove.add_argument('new_file', type=str, help='')
+        parser_remove.set_defaults(func=self.remove)
+
+        # assemble command args
+        parser_assemble = subparsers.add_parser('assemble')
+        parser_assemble.add_argument('guid', type=str, help='guid')
+        parser_assemble.add_argument('file_type', type=str, help='')
+        parser_assemble.add_argument('comp', type=str, help='')
+        parser_assemble.add_argument('raw_file', type=str, help='')
+        parser_assemble.add_argument('efi_file', type=str, help='')
+        parser_assemble.set_defaults(func=self.assemble)
+
+        parser.parse_args(self.argv[2:], namespace=self)
+
         # No driver required when printing the util documentation
```

Comment:
```suggestion
        # No driver for decode functionality
        if self.argv[2] in ('decode'):
            return False
        return True
```
Believe that 181 - 191 can be simplified to the above